conceptnet5/vectors/debias.py

```diff
@@ -394,10 +394,9 @@ def reject_subspace(frame, vecs):
     """
     current_array = frame.copy().values
     for vec in vecs:
-        if not np.isnan(vec).any():
```

Comment:
Because the thing that could have called it with NaN values was:

    components_to_reject = frame.loc[vocab].values

on line 531. But now we use `reindex()` and `dropna()` there.